Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35057&attempt_id=6fccfe14-97cc-479e-8ea9-5c397244159b)

<details><summary>Partial log (click to expand)</summary><p>

```
                 from qmc2main.cpp:43:
joystick.h:11:10: fatal error: SDL.h: No such file or directory
 #include <SDL.h>
          ^~~~~~~
compilation terminated.
make[1]: *** [Makefile.qmake:2888: qmc2main.o] Error 1
make[1]: Leaving directory '/tmp/nix-build-qmc2-0.195.drv-0/qmc2'
make: *** [Makefile:1043: qmc2-bin] Error 2
builder for ‘/nix/store/hqzqvpdj7yq51719zqm8ik5iifzfnbap-qmc2-0.195.drv’ failed with exit code 2
error: build of ‘/nix/store/hqzqvpdj7yq51719zqm8ik5iifzfnbap-qmc2-0.195.drv’ failed
```
</p></details>

--- a/pkgs/misc/emulators/qmc2/default.nix
+++ b/pkgs/misc/emulators/qmc2/default.nix
@@ -1,0 +1,40 @@
+{ stdenv
+, fetchurl, qmake, qttools, pkgconfig
+, minizip, zlib
+, qtbase, qtsvg, qtmultimedia, qtwebkit, qttranslations, qtxmlpatterns
+, rsync, SDL2, xwininfo
+, utillinux
+, xorg
+}:
+
+stdenv.mkDerivation rec {
+  name = "qmc2-${version}";
+  version = "0.195";
+
+  src = fetchurl {
+      url = "mirror://sourceforge/project/qmc2/qmc2/${version}/${name}.tar.gz";
+      sha256 = "1dzmjlfk8pdspns6zg1jmd5fqzg8igd4q38cz4a1vf39lx74svns";
+  };
+  
+  preBuild = ''
+    patchShebangs scripts
+  '';
+
+  nativeBuildInputs = [ qttools pkgconfig ];
+  buildInputs = [ minizip qtbase qtsvg qtmultimedia qtwebkit
+                  qttranslations qtxmlpatterns rsync SDL2
+                  xwininfo zlib utillinux xorg.libxcb ];
+
+  makeFlags = [ "DESTDIR=$(out)"
+                "PREFIX=/"
+                "DATADIR=/share/"
+                "SYSCONFDIR=/etc" ];
+
+  meta = with stdenv.lib; {
+    description = "A Qt frontend for MAME/MESS";
+    homepage = https://qmc2.batcom-it.net;
+    license = licenses.gpl2;
+    maintainers = [ maintainers.genesis ];
+    platforms = platforms.linux;
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -14300,6 +14300,8 @@ with pkgs;
 
   qgo = libsForQt5.callPackage ../games/qgo { };
 
+  qmc2 = libsForQt5.callPackage ../misc/emulators/qmc2 { };
+
   quattrocento = callPackage ../data/fonts/quattrocento {};
 
   quattrocento-sans = callPackage ../data/fonts/quattrocento-sans {};

